native/src/main/java/io/ballerina/lib/ldap/ModuleUtils.java

```diff
@@ -39,11 +39,11 @@ public final class ModuleUtils {
     public static final String LDAP_RESPONSE = "LdapResponse";
     public static final String RESULT_STATUS = "resultCode";
     public static final String MATCHED_DN = "matchedDN";
+    public static final String DIAGNOSTIC_MESSAGE = "diagnosticMessage";
+    public static final String REFERRAL = "referral";
     public static final String OPERATION_TYPE = "operationType";
```

Comment:
Better to move these constants to the class where these values are used, unless they are used in multiple places.